EAP7-881.adoc

```diff
@@ -0,0 +1,88 @@
+= Provide launch.openshift.io experience
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          openshift,launch,developer
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-881
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+This support is provided by Fabric8 launcher: https://github.com/fabric8-launcher.
+
+=== Other Interested Projects
+
+== Requirements
+
+ * provide generators for creation of new applications with WildFly
```

Comment:
I can see WildFly option at https://launch.openshift.io/launch - is the WF part done already or does this cover something else?